android-components/components/browser/storage-sync/src/test/java/mozilla/components/browser/storage/sync/PlacesHistoryStorageTest.kt

```diff
@@ -545,9 +544,9 @@ class PlacesHistoryStorageTest {
         assertEquals(0, visits.size)
     }
 
-    @Ignore("Disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1853687")
     @Test
     fun `store can delete by 'range'`() = runTestOnMain {
+        history.writeScope = this
```

Comment:
Drive-by 🚗 : Should we do this in the setup so it's not a footgun for future tests?

Reply:
I can file a follow up, but I wasn't sure without a deeper investigation whether the test scope for setup would carry over to the test function itself

Reply:
This should allow us to test whether this fixes the flaky behavior at least. If it does I will move forward with your suggestion, thanks!

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1858240